src/main/java/edu/ucsb/cs156/happiercows/controllers/CommonsController.java

```diff
@@ -117,11 +109,46 @@ public Commons getCommonsById(
   public ResponseEntity<Commons> deleteUserFromCommon(@PathVariable("commonsId") Long commonsId,
       @PathVariable("userId") Long userId) throws Exception {
 
+
     UserCommons uc = userCommonsRepository.findByCommonsIdAndUserId(commonsId, userId)
       .orElseThrow(
           () -> new EntityNotFoundException(UserCommons.class, "commonsId", commonsId, "userId", userId));;
       
     userCommonsRepository.deleteById(uc.getId());
+
     return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
   }
-}
\ No newline at end of file
+  
+  @ApiOperation("Delete a commons by id")
+  @PreAuthorize("hasRole('ROLE_ADMIN')")
+  @DeleteMapping("delete")
+  public ResponseEntity<String> deleteCommons(
+    @ApiParam("id of common to delete") @RequestParam Long id) throws JsonProcessingException {
+      Commons commons = commonsRepository.findById(id)
+        .orElseThrow(() -> new EntityNotFoundException(Commons.class, id));
+
+      commonsRepository.deleteById(id);
+      return ResponseEntity.ok().body(String.format("record %d deleted", id));
+    }
```

Comment:
I'll let this slide since it's the last day, but I think we need a 5 point issue to convert this to this format:

Right now, this controller returns a string `"record 17 deleted"` for example.

A preferred way is now shown here:

https://github.com/ucsb-cs156-w22/demo-spring-react-example-v2/blob/59bed32913f809dbcd40a55e7caa3d79b39daabc/src/main/java/edu/ucsb/cs156/example/controllers/TodosController.java#L125

Where you return an `Object` instead of a `ResponseEntity<String>` that is formatted JSON.

Reply:
Here's an issue ....
https://github.com/ucsb-cs156-w22/team04-w22-5pm-HappyCows/issues/53